### DISCUSSION TITLE
Add 'emergency=defibrillator' to 'Is this still here?' quest

### DISCUSSION THREAD

**Discussion by ERYpTION** [streetcomplete/StreetComplete#4094] [Ideas]
As far as I can tell defibrillators are not included in the quest type 'Is this still here?'. I would suggest a resurvey interval of 6 months.

**Comment by FloEdelmann** [comment]
It is asked for outdoor defibrillators:
https://github.com/streetcomplete/StreetComplete/blob/a2d9b8c9bf7046825a5b33f4d3c712b55723ee7a/app/src/main/java/de/westnordost/streetcomplete/quests/existence/CheckExistence.kt#L45

See https://github.com/streetcomplete/StreetComplete/issues/2744#issuecomment-819433429.
